### ISSUE TITLE
[Question] Resources on how to make plugins

### ISSUE THREAD

**Issue by snow-blade** [rxi/lite#178]
Hi, i've been using lite for over a month and i would like to know some good resources on how i can make plugins to level-up my experience with this awesome text editor, i've been searching around trying to see how other plugins work but in the end i always hit a wall when i try to implement those myself, making the colorschemes is pretty easy but for the rest i would like to know some resources which help me interact with the API, also do you plan on making some kind of wiki for the API, if so it would really be useful,<br>
Keep up the good work

**Comment by Tmpod** [comment]
Yeah this is something that unfortunately hasn't been taken care of. In my experience, the best resources are really the already existing plugins and lite's source code. Once you understand how the project is structured, it becomes fairly easy to navigate it and explore the API it "provides". I am starting to work on documenting the whole codebase but it's a ton of work and won't be ready soon, as it's also not a very high-priority thing for me atm. 

Can't say much more than this xD  
I guess you can check my plugin collection (https://sr.ht/~tmpod/lite-plugins) to which I have been progressively adding more stuff to, and you can also hit me up on Discord (Tmpod#2525) or Matrix (@tmpod:matrix.org) if you want to have a more direct chat ;)

**Comment by snow-blade** [comment]
wow, Thanks for taking your time and reading this old  issue of mine, i do not use lite as often as i used to do because this issue was not answered, i think i will re look into lite if i could get some decent documentation or easier examples
